### PR TITLE
Implementing app navigation

### PR DESCRIPTION
**Pull request by ichandel**
Finished setting up app nav, took a while due to having to deal with our previously integrated code. As of right now, data transfer between fragments isn't set up but shouldn't be too difficult for those working on those branches to implement. Please let me know if you run into any bugs so we can solve them together! o7

### CODE REVIEW

**Review by aurbeez — DISMISSED** [review]
looks good to me so far after running it and looking into viewmodels a bit. Don't fully understand viewmodels but if I run into anything while implementing my stories with this, I'll let you know :v)

File: src/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.MyApplication"
+        android:theme="@style/Theme.MyApplication."
```
Comment (Cedric-Boucher) [inline src/app/src/main/AndroidManifest.xml:13]: what did this change?

File: src/app/src/main/java/com/example/myapplication/Facility.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
 package com.example.myapplication;
 
+import androidx.fragment.app.Fragment;
```
Comment (Cedric-Boucher) [inline src/app/src/main/java/com/example/myapplication/Facility.java:3]: I assume this was accidental?

**Review by Cedric-Boucher — DISMISSED** [review]
looks good. I didn't look in detail into any of the fragment or xml stuff. I left a couple comments as well

File: src/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (ichandel) [reply src/app/src/main/AndroidManifest.xml:13]: our style theme name had a period in it so I was just making it the same for continuity sake

File: src/app/src/main/java/com/example/myapplication/Facility.java (hunk 2 above)
Comment (ichandel) [reply src/app/src/main/java/com/example/myapplication/Facility.java:3]: Yes, thank you for catching it. I'll update it in main unless there's a way to fix rn?

**Review by aurbeez — APPROVED** [review]
cooked to perfection

**Review by Ociliv1 — APPROVED** [review]

**Pull request merged**